deploy/examples/cluster-test.yaml

```diff
@@ -55,7 +54,7 @@ spec:
 apiVersion: ceph.rook.io/v1
 kind: CephBlockPool
 metadata:
-  name: mgr
+  name: builtin-mgr
```

Comment:
changed this to `builtin-mgr` to reflect the `.nfs` example, but this is not a critical change.